miner_objects/position_inspector.py

```diff
@@ -38,7 +39,12 @@ def get_recently_acked_validators(self):
     def get_possible_validators(self):
         # Right now bittensor has no functionality to know if a hotkey 100% corresponds to a validator
         # Revisit this in the future.
-        return self.metagraph.axons
+        if not self.is_testnet:
```

Comment:
nit: make code more concise by removing the "not" and flipping the conditional code block locations